src/main/java/cms/gov/madie/measure/resources/MeasureController.java

```diff
@@ -116,6 +124,7 @@
         measure.setCreatedBy(persistedMeasure.get().getCreatedBy());
         repository.save(measure);
         response = ResponseEntity.ok().body("Measure updated successfully.");
+        //        log.info("Response.entity [{}]", ResponseEntity);
```

Comment:
Was this log unhelpful?

Reply:
Left over failed log attempt towards a class